perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -112,32 +117,28 @@ def _export_to_gsheet(folders, metrics, command, worksheet) -> None:
         metrics[testing_folder.name]['Quantiles']['99.9 %ile'],
         metrics[testing_folder.name]['Quantiles']['100 %ile']
     ]
-    gsheet_data.append(row)
+    # Only some extracted metrics will be uploaded to Google Spreadsheets and BigQuery
+    row_data_to_upload = row[1:3] + [row[4]] + [row[8]] + [row[17]] + row[5:8] + row[9:13]
```

Comment:
Instead of selecting some values in 'row_data_to_upload', can we simply not add them in the 'row' at first place ? (Given we are anyways changing the schema of gsheet ?)